### ISSUE TITLE
Issue with ad_group_ad.policy_summary field

### ISSUE THREAD

**Issue by mortiy** [Opteo/google-ads-api#67]
### I'm submitting a ...

-   [x] bug report
-   [ ] feature request
-   [ ] question about the decisions made in the repository
-   [ ] question about how to use this project

### Summary

The `policy_summary` in `AdGroupAd` resolved to `undefined`.
While debugging protobufs I see the data for this attribute, but it's lost somewhere during converting to JSON.

https://developers.google.com/adwords/api/docs/reference/v201809/AdGroupAdService.AdGroupAdPolicySummary

### Other information
```typescript
return client
  .report({
    entity: 'ad_group_ad',
    attributes: ['ad_group_ad.status', 'ad_group_ad.policy_summary'],
    constraints
  })
```

**Comment by avermeil** [comment]
Hi @mortiy -- Thanks for the bug report! Sorry about taking so long to handle this.

We've got a fix ready, which we'll release in a new version soon. (https://github.com/Opteo/google-ads-node/commit/ebad895a3cbc2041c6fadc88ba02cb8869de01a2)

**Comment by avermeil** [comment]
Fixed in v3.2.0. Let us know if you have any more issues with this :)
